inst/templates/config-template.txt

```diff
@@ -8,8 +8,14 @@
 # lc: Library Carpentry
 # cp: Carpentries (to use for instructor training for instance)
 # incubator: The Carpentries Incubator
+# Note that you can also use a custom carpentry type. For more info,
+# see the documentation: https://carpentries.github.io/sandpaper-docs/editing.html
 carpentry: {{ carpentry }}
 
+# Custom carpentry description
+# This will be used as the alt text for the logo
+# carpentry_description: "Custom Carpentry"
```

Comment:
Note to self that this change will need to be reflected in the two template repositories, https://github.com/carpentries/workbench-template-md and https://github.com/carpentries/workbench-template-rmd